enterprise/server/auth/auth.go

```diff
@@ -705,6 +709,7 @@ func authContextFromClaims(ctx context.Context, claims *Claims, err error) conte
 		return AuthContextWithError(ctx, err)
 	}
 	ctx = context.WithValue(ctx, contextTokenStringKey, tokenString)
+	ctx = context.WithValue(ctx, contextClaimsKey, claims)
```

Comment:
It would be nice if we could store a single value (struct) in the context that contains both the jwt and the claims information. That would prevent them from getting out of sync as the auth code evolves in the future. It would be a more invasive change so no need to do it in this PR, but I think it would be good to follow up on it or file an issue for the future.

Reply:
+1